src/pages/cart/Services.js

```diff
@@ -40,8 +40,11 @@ function Services({ couponid }) {
   const [likeCount, setLikeCount] = useState(0);
   const [dislikeCount, setDislikeCount] = useState(0);
   const [activeBtn, setActiveBtn] = useState("none");
+  const [getwishdata ,setWishData] = useState('')
   const Data = useSelector((state) => state.saloonData);
   const whishdata = useSelector((state) => state.whishlistItem);
+  // console.log('oooohdfhfxdhdfh',whishdata)
+  console.log('getwishdatagetwishdata',getwishdata)
```

Comment:
remove all the consoles whenever you pushed your code

Reply:
okk mam